src/strategies/views/home-view.test.ts

```
@@ -0,0 +1,30 @@
+
+vi.mock('@smolpack/hasskit', () => ({
+  Home: class {
+    floors = [];
+    weatherEntity = undefined;
+    climateEntity = undefined;
+    lightEntity = undefined;
+    lockEntity = undefined;
+    mediaPlayerEntity = undefined;
+    co2SignalEntity = undefined;
+    wasteEntity = undefined;
+    constructor(_hass: unknown, _config?: unknown) {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)